install/kubernetes/cilium/charts/hubble-ui/templates/servicemonitor.yaml

```diff
@@ -0,0 +1,21 @@
+{{- if and .Values.metrics.enabled (.Values.metrics.serviceMonitor.enabled) }}
```

Comment:
This is not related to ui, therefore should not be part of the `hubble-ui` chart. Not sure what do with it though, maybe move it into the `agent` chart for now? Or create a new `metrics` chart.

Reply:
For now I moved it to the cilium-agent servicemonitor. Need to check on that metrics option, and see if we have an equivalent one in the values.yaml or otherwise remove the check. Working on it.

Reply:
Addressed review comments.